src/undo.h

```diff
@@ -193,9 +175,10 @@ class CBlockUndo
     bool includesSidechainAttributes;
 
 public:
-    std::vector<CTxUndo> vtxundo; // for all but the coinbase
+    std::vector<CTxUndo>                vtxundo;         // for all txs and certs but the coinbase
     uint256 old_tree_root;
-    std::map<uint256, ScUndoData> msc_iaundo; // key=scid, value=amount matured at block height
+    std::vector<CVoidedCertUndo> vVoidedCertUndo; // for all but the coinbase
```

Comment:
just to be formally consistent with the serialization order they could be swapped

Reply:
Done